core/src/main/scala/org/apache/spark/sql/eventhubs/EventHubsSourceProvider.scala

```diff
@@ -62,14 +61,7 @@ private[sql] class EventHubsSourceProvider
     EventHubsClient.userAgent =
       s"Structured-Streaming-${sqlContext.sparkSession.sparkContext.version}"
 
-    val caseInsensitiveParameters = parameters.map {
```

Comment:
What is the thought process for removing these? I'm pretty sure all the lookups assume lowercase keys.

Reply:
I would assume that people would supply the parameters in lowercase anyway. This logic might add a big of unpredictable behaviour. I would suggest to put it here in the factory, to keep everything on a single place:
https://github.com/Fokko/azure-event-hubs-spark/blob/fd-fix-event-hub-offset/core/src/main/scala/org/apache/spark/sql/eventhubs/EventHubsSourceProvider.scala#L147

What do you think?

Reply:
I've moved all the `.toLowerCase()` to the `EventHubsConf` to keep everything in one single place 👍

Reply:
Ok, cool - that's definitely the right move.